server/server.go

```
@@ -72,9 +149,94 @@ func (n *Node) Start() {
+	}
+
+	ctx := context.Background()
+	response, _ := n.AppendEntries(ctx, &raftpb.AppendRequest{
+		Term:         int32(n.RaftNode.GetCurrentTerm()),
+		LeaderId:     n.id,
+		Entries:      req.Entries,
+		LeaderCommit: int32(n.RaftNode.GetCommitIndex()),
+	})
+
+	w.Header().Set("Content-Type", "application/json")
+	json.NewEncoder(w).Encode(&pb.AppendResponse{
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)